website/content/api-docs/operator/raft.mdx

```diff
@@ -100,18 +130,100 @@ The table below shows this endpoint's support for
 
 ### Parameters
 
-- `address` `(string: <optional>)` - Specifies the server to remove as
-  `ip:port`. This cannot be provided along with the `id` parameter.
+- `address` `(string: <optional>)` - Specifies the Raft **Address** of the
+  server to remove as provided in the output of `/v1/operator/raft/configuration`
+  API endpoint or the `nomad operator raft list-peers` command.
 
-- `id` `(string: <optional>)` - Specifies the server to remove as
-  `id`. This cannot be provided along with the `address` parameter.
+- `id` `(string: <optional>)` - Specifies the Raft **ID** of the server to
+  remove as provided in the output of `/v1/operator/raft/configuration`
+  API endpoint or the `nomad operator raft list-peers` command.
+
+  <Note>
+
+  Either `address` or `id` must be provided, but not both.
+
+  </Note>
 
 ### Sample Request
 
+
+<Tabs>
+<Tab heading="Nomad CLI">
```

Comment:
Leave this for now, but adding a brand new way to demonstrate API examples is the sort of thing that really doesn't belong in a feature PR. We end up with a bunch of one-off decisions that way that aren't applied equally across the API docs. In the future, if you want to do this sort of thing, make a separate PR for it so that we can discuss it independently of critical feature work.